zachang/HelloBooks
# #153906204 users should be able to modify their profile…

#### What does this PR do?
Implements modify profile feature.

#### Description of Task to be completed?
It makes it possible for users to modify their profile on the app.

#### How should this be manually tested?
+ Users should login then click the `profile link` to navigate to the profile page, then modify their profile if need be. 

#### What are the relevant pivotal tracker stories?
`#153906204`

#### Questions:
None
